buildSrc/src/main/java/com/blackduck/integration/detect/docs/content/Terms.java

```diff
@@ -10,10 +10,12 @@ public Terms() {
         termMap.put("solution_name", "Black Duck Detect");
         termMap.put("script_repo_url_bash", "https://detect.blackduck.com/detect10.sh");
         termMap.put("script_repo_url_powershell", "https://detect.blackduck.com/detect10.ps1");
-        termMap.put("binary_repo_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/detect");
-        termMap.put("binary_repo_ui_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/detect");
-	termMap.put("binary_repo_jenkins_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackducksoftware/integration/blackduck-detect/");
-	termMap.put("binary_repo_url_sigma", "https://repo.blackduck.com/sigma-release-trial/2022.6.0/");
+        termMap.put("binary_repo_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/detect/");
+        termMap.put("binary_repo_ui_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/detect/");
+		//No UI for repo.blackduck.com as of 2024-09-25
+		termMap.put("binary_repo_jenkins_url_project", "https://repo.blackduck.com/bds-integrations-release/com/blackducksoftware/integration/blackduck-detect/");
```

Comment:
nit: would be nice to not mix spaces and tabs

Reply:
True. Will fix that.